src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -537,15 +537,19 @@ function enableNewQuestion() {
 
     moveAssignWeightsCheckbox($newQuestionTable.find(`#rubricAssignWeights-${NEW_QUESTION}`));
 
-    if ($(`#generateOptionsCheckbox-${NEW_QUESTION}`).prop('checked')) {
+    if ($(`#generateMcqOptionsCheckbox-${NEW_QUESTION}`).prop('checked')) {
```

Comment:
After you have added this, we will have three (almost) identical code in `disableQuestion` `enableQuestion` and `enableNewQuestion`. See if you can extract these lines into a common method.

Reply:
It's possible to extract from `enableQuestion` and `enableNewQuestion` only.

The other one although possible will result in extra code in the function which won't be used by other two calls. Also the logic isn't exactly the same, so thee will still be code duplication

Reply:
@tshradheya See `toggleMcqGeneratedOptions()` in `questionMcq.js`. You can make use of that method actually instead of writing it again.